packages/graph/migrations/postgres/migration/1656417312397_initial.ts

```diff
@@ -282,12 +282,17 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
     {
       entity_id: {
         type: "UUID",
-        primaryKey: true,
+        notNull: true,
+      },
+      version: {
+        type: "TIMESTAMP WITH TIME ZONE",
```

Comment:
We don't store the time zone. This only affects the way how the data is inserted. WITH TIME ZONE will adjust the timestamp inserted to UTC.

For further reading, see:

- https://www.postgresql.org/docs/current/datatype-datetime.html#DATATYPE-TIMEZONES
- https://stackoverflow.com/a/5876276/3131852